src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -174,6 +174,12 @@ public ITaskItem[] References
             get { return (ITaskItem[])_store["References"]; }
         }
 
+        public bool ReportAnalyzer
```

Comment:
I don't think so. but this will go for RTM so, I think by RTM, task will be moved to Roslyn I believe, @agocke ?